bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/i18n/omnilink.properties

```diff
@@ -318,3 +318,10 @@ channel-type.omnilink.zone_latched_alarm_status.state.option.1 = Tripped
 channel-type.omnilink.zone_latched_alarm_status.state.option.2 = Reset, but previously tripped
 channel-type.omnilink.zone_restore.label = Restore Zone
 channel-type.omnilink.zone_restore.description = Send a 4 digit user code to restore this zone.
+
+# thing actions
+
+actionInputZoneLabel = Time zone
+actionInputZoneDesc = The time zone of the controller, provided in the "America/Denver" format.
+actionLabel = Synchronize the Date/Time and DST flag of the controller
```

Comment:
Can you shorten the label? Perhaps simply "Synchronize controller Date/Time" and let the description supply the detailed version.